learning/features/math/vector_math.rst

```diff
@@ -123,7 +123,7 @@ Let's look at two common uses for vector addition and subtraction.
 
 - Movement
 
-A vector can represent **any** quantity with a magnitude and direction. Typical examples are: position, direction, velocity, acceleration, force etc. In
+A vector can represent **any** quantity with a magnitude and direction. Typical examples are: position, velocity, acceleration and force. In
```

Comment:
Missing an Oxford comma here. Our doc standards are to use it: "acceleration, and force."